.git-blame-ignore-revs

```diff
@@ -0,0 +1,11 @@
+#################
```

Comment:
nit: add some comments here to illustrate the usage of this file
I just borrow the content from [`pydata/pydata-sphinx-theme`](https://github.com/pydata/pydata-sphinx-theme/pull/713/files)

```
# When making commits that are strictly formatting/style changes, add the
# commit hash here, so git blame can ignore the change.
#
# For more details, see:
# https://git-scm.com/docs/git-config#Documentation/git-config.txt-blameignoreRevsFile
#
```

Reply:
modified, thx